.github/workflows/create-dev-tag.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```

Comment:
You can upgrade to v0.9.0 as well :)

Reply:
https://github.com/mesosphere/kommander-applications/pull/2100/commits/b8c930ac5feb6068de25c119a978db7e2ddbfe89